client/state/products-list/selectors.js

```diff
@@ -110,5 +109,5 @@ export const computeProductsWithPrices = ( state, siteId, planSlugs ) => {
 			...computeFullAndMonthlyPricesForPlan( state, siteId, availablePlanProduct.plan ),
 		} ) )
 		.filter( availablePlanProduct => availablePlanProduct.priceFull )
-		.sort( ( a, b ) => TERMS_SORTED.indexOf( b.plan.term ) - TERMS_SORTED.indexOf( a.plan.term ) );
+		.sort( ( a, b ) => getTermDuration( b.plan.term ) - getTermDuration( a.plan.term ) );
```

Comment:
hm. okay well I think this makes more sense when reading it. just curious - why don't we store the duration on the plan object itself?

Reply:
We could but we already store term so it would require keeping the two properties in sync.

Reply:
Is it important that the constants `TERM_MONTHLY` and others are strings? Could we actually store them as the values of the durations?